generators/sram-bist/src/test/scala/BistTopSpec.scala

```diff
@@ -474,11 +474,9 @@ class BistTopSpec extends AnyFlatSpec with ChiselScalatestTester {
           )
         )
       )(
-        new WithChiseltestSrams(ChiseltestSramFailureMode.none)
+        new freechips.rocketchip.subsystem.WithClockGateModel
```

Comment:
This should be with Chiseltest SRAMs, the clock gate model shouldn't be needed for chiseltest.